wp-content/civi-extensions/goonjcustom/goonjcustom.php

```diff
@@ -277,4 +300,43 @@ function goonjcustom_civicrm_tabset($tabsetName, &$tabs, $context) {
 		'active' => 1,
 		'current' => false,
 	];
+
+	// Add JavaScript to dynamically hide fields
+	CRM_Core_Region::instance('page-footer')->add(
```

Comment:
Is there a check we can add to load this JS only the page where we need it?

Reply:
yes